ui/src/components/WebRTCVideo.tsx

```diff
@@ -667,6 +550,18 @@ export default function WebRTCVideo() {
         };
   }, [videoSaturation, videoBrightness, videoContrast]);
 
+  function getAdjustedKeyCode(e: KeyboardEvent) {
```

Comment:
I'm not sure this is still needed, but if it _is_ then we have to do it for both key down and key up... so